### PR TITLE
[Docs] Add docstring for `KafkaTarget`

### PR DESCRIPTION
**Pull request by gtopper**
Based on #5380.

### CODE REVIEW

File: mlrun/datastore/targets.py [hunk 1]
```diff
@@ -1516,6 +1516,23 @@ def as_df(self, columns=None, df_module=None, **kwargs):
...
+    example::
+        # define target
+        kafka_target = KafkaTarget(
+            name="kafka", path="my_topic", brokers="localhost:9092"
+        )
+        # ingest
+        stocks_set.ingest(stocks, [kafka_target])
+    :param name:                target name.
+    :param path:                topic name e.g. "my_topic"
+    :param bootstrap_servers:   Deprecated. Use the brokers parameter instead.
```
Comment (assaf758) [inline mlrun/datastore/targets.py]: Since it's deprecated, to ease readability, please move below "brokers"

File: mlrun/datastore/targets.py [hunk 2]
```diff
@@ -1516,6 +1516,23 @@ def as_df(self, columns=None, df_module=None, **kwargs):
...
+    :param producer_options:    additional configurations for kafka producer
+    :param brokers:             kafka broker, or list of kafka brokers, e.g. "localhost:9092", or
+        ["kafka-broker-1:9092", "kafka-broker-2:9092"]
```
Comment (assaf758) [inline mlrun/datastore/targets.py]:
```suggestion
    :param brokers:             kafka broker url (e.g. "127.0.0.1:9092"), or a list of kafka broker urls, 
       (e.g ["kafka-broker-1:9092", "kafka-broker-2:9092"])
```

**Review by assaf758 — CHANGES_REQUESTED** [review]

File: mlrun/datastore/targets.py (hunk 1 above)
Comment (gtopper) [reply mlrun/datastore/targets.py]: The docstring order should match the actual parameter order... Which can't be changed for bwc.

File: mlrun/datastore/targets.py (hunk 2 above)
Comment (gtopper) [reply mlrun/datastore/targets.py]: Those are not URLs.

**Review by assaf758 — APPROVED** [review]

**Pull request merged**